cfme/fixtures/utility_vm.py

```diff
@@ -46,9 +45,11 @@ def connect_ssh(vm, creds):
         try:
             with SSHClient(hostname=ip, username=creds.principal, password=creds.secret) as client:
                 logger.info("SSH connected to IP %s", ip)
+
                 result = client.run_command("true")
                 if not result.success:
                     raise Exception(f"Command `true` failed on ip {ip}.")
+
                 yield client
                 return
         except Exception as ex:
```

Comment:
The Exception raised for the `true` command failure makes the meassage bit misleading.

Reply:
Ah. I see you removed the return. That was confusing me but I didn't notice you had removed it. I was seeing that only the first IP was getting used.

Reply:
Why are we yielding and returning on lines 52, 53 above? Should't we just return a client?